### PR TITLE
Remove promxy resource

### PR DESCRIPTION
**Pull request by QuentinBisson**
Remove promxy resource not needed after  https://github.com/giantswarm/promxy-app/pull/33
## Checklist

I have:

- [x] Described why this change is being introduced
- [ ] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

**Comment by TheoBrigitte** [comment]
Is it safe to just delete the resource? Shouldn't we convert it to a delete resource first ?

**Comment by QuentinBisson** [comment]
We could but deploying the promxy app in the newer version should override the configmap anyway

**Comment by QuentinBisson** [comment]
My idea for next week is to actually release pmo without the promxy resource then promxy

**Comment by TheoBrigitte** [comment]
Fine we me then when the config gets overwritten then, I just wanted to make sure we are not leaving anything behind to be cleaned up.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Split `ManagementClusterCertificateWillExpireInLessThanTwoWeeks` alert per provider.
 
+### Removed 
+
+Remove promxy resource.
```
Comment (JosephSalisbury) [inline CHANGELOG.md:27]:
```suggestion
- Remove promxy resource.
```

**Review by JosephSalisbury — APPROVED** [review]

**Pull request merged**